website/integrations: Update Semaphore UI description (#12674)
goauthentik/authentik: website/integrations/services/semaphore/index.mdx

## authentik configuration

Start the wizzard for adding a new application.

**1. Application:**

- Name: `Semaphore UI`
- Slug: `semaphore`

**2. Choose a Provider**

Select `OAuth2/OpenID Provider`

**3. Configure Provider**

Select implicit or explicit authorization flow as desired.

Take note of the Client ID and Client Secret, you'll need to give them to Semaphore UI later.

- Redirect URIs/Origins (RegEx): `https://semaphore.company/api/auth/oidc/authentik/redirect/`
- Signing Key: `authentik Self-signed Certificate`

Leave the rest as default values.

## Semaphore UI configuration

> Log in to your Semaphore UI host via SSH. Edit the `/etc/semaphore/config.json` file with the text editor of your choice. 

Author: ```suggestion
Log in to your Semaphore UI host via SSH. Edit the `/etc/semaphore/config.json` file with the text editor of your choice.
```

Add the `oidc_providers` configuration:

```
{
  "oidc_providers": {
    "authentik": {
      "display_name": "Sign in with Authentik",
      "provider_url": "https://authentik.company/application/o/<slug>/",
      "client_id": "<client-id>",
      "client_secret": "<client-secret>",
      "redirect_url": "https://semaphore.company/api/auth/oidc/authentik/redirect/",
      "username_claim": "username",
      "name_claim": "name",
      "email_claim": "email",
      "scopes": ["openid", "profile", "email"]
  },
  ...
}
```

:::info
The name of the oidc_provider (e.g. `authentik`) need to match the name on the redirect url.
:::

:::info

> If there is a `Not Found` error shown after the login and redirect to `https://semaphore.company/api/auth/oidc/authentik/`, it can help to set the web_root to `/`: 

Reviewer: ```suggestion
If  a `Not Found` error is displayed after the login, and you are redirected to `https://semaphore.company/api/auth/oidc/authentik/`, you might need to set the web_root to `/`: 
```

Author: ```suggestion
If a `Not Found` error is displayed after the login, you might need to set the web_root to `/` (see https://github.com/semaphoreui/semaphore/issues/2681): 
```
